### ISSUE TITLE
Fairness and proxy-to-origin

### ISSUE THREAD

**Issue by LPardue** [httpwg/http-extensions#1368]
The I-D contains the following TODO

> TODO: Discuss if we should add a signal that mitigates this issue. For example,
we might add a SETTINGS parameter that indicates the next hop that the
connection is NOT coalesced (see https://github.com/kazuho/draft-kazuho-httpbis-priority/issues/99).

I'd like to track that discussion here. Namely, with all the changes in the meantime, does @rmarx think that there is still a problem that needs fixing? OR can we close with no action?

**Comment by rmarx** [comment]
I don't think anything additional is needed. We've got the two standard header options to indicate an intermediary is involved, that was the main point of the original discussion in Singapore IIRC. 

I think this won't matter all that much in practice, given warmed up edge-to-origin connections, pops being able to serve cached resources in between, etc. Origin owners really concerned with performance will know if their setup is coalescing or not (then the question is if origin server software allows tweaking prioritization behavior, but that's a general concern). CDN owners seeing bad origin behaviour will probably just overwrite the priority information instead of forwarding in practice. 

In short, I think the current text `if a server knows the intermediary is coalescing requests` should suffice here, as I don't think many servers would support switching to RR when seeing an explicit "warning: coalescing intermediary" signal either way.

**Comment by LPardue** [comment]
Thanks @rmarx. In that case I've created #1369 to simply remove the TODO text, which will act to resolve this issues.
